design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,199 @@
+# Ignore changes
+
+* Owners: Lovro Sviben (@lsviben)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Background
+
+When Crossplane manages a resource, it reconciles all of the parameters under `spec.forProvider` to the external providers both 
+during the creation and subsequent updates to the managed resource. But in some cases, a parameter of the external resource
+could change due to it being managed outside of Crossplane. In the current state, if that happens, Crossplane will try to 
+"fix" the situation, and update the parameter back to the one it has in the `spec.forProvider` as it is designed to be
+the only source of truth.
+
+Some examples of this case:
+- [AutoScalingGroup DesireCapacity], where when using an autoscaler, the parameter gets changed 
+externally.
+- [DynamoDB Tables] with using autoscaling ([Policy] + [Target]). The autoscaler scales the read and write capacity,
+and then Crossplane resets it back to the `readCapacity` and `writeCapacity` defined on the Table, which effectively makes autoscaling useless.
+- [NodeGroup] while using a node autoscaler. There is a race condition between crossplane and node autoscaler. 
+Node autoscaler increases the `desiredSize` due to resource requirement but crossplane decrease it to `desiredSize` set in composition.
+
+In these cases, it would make sense to make Crossplane ignore select parameters during updates. 
+Terraform addresses this use case with its [ignore_changes] functionality. 
+
+An additional benefit of ignoring some fields would be that it would be possible to have different Crossplane
+Compositions managing a single external resource. This could open up some interesting new possibilities for Crossplane.
+
+So in this document, the aim will be to introduce a solution in Crossplane for ignoring some managed resource parameters
+during updates.
+
+## Goals
+
+* Design a functionality that allows for ignoring some fields during resource updates.
+* Try and make it as easy as possible to apply the ignoring functionality on existing Crossplane providers. In other words, to 
+design a standard way on how to update the providers code to support `ignore changes`.
```

Comment:
I think _a standard way_ is more like one of the solutions rather than a goal we should aim for. There might be solutions that are not shared across all MRs that will achieve the same goals.

Reply:
Right, removed the goal as its vague